plugins/editors/tinymce/src/PluginTraits/GlobalFilters.php

```diff
@@ -0,0 +1,207 @@
+<?php
+/**
+ * @package     Joomla.Plugin
+ * @subpackage  Editors.tinymce
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+namespace Joomla\Plugin\Editors\TinyMCE\PluginTraits;
+
+\defined('_JEXEC') or die();
+
+use Joomla\CMS\Access\Access;
+use Joomla\CMS\Component\ComponentHelper;
+use Joomla\CMS\Filter\InputFilter;
+
+/**
+ * Handles the Joomla filters for the TinyMCE editor.
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+trait GlobalFilters
+{
+	/**
+	 * Get the global text filters to arbitrary text as per settings for current user groups
+	 * @param   User  $user  The user object
+	 *
+	 * @return  JFilterInput
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	protected static function getGlobalFilters($user)
+	{
+		// Filter settings
+		$config     = ComponentHelper::getParams('com_config');
+		$userGroups = Access::getGroupsByUser($user->get('id'));
+		$filters    = $config->get('filters');
+
+		$forbiddenListTags       = [];
+		$forbiddenListAttributes = [];
+		$customListTags          = [];
+		$customListAttributes    = [];
+		$allowedListTags         = [];
+		$allowedListAttributes   = [];
+
+		$allowedList   = false;
+		$forbiddenList = false;
+		$customList    = false;
+		$unfiltered    = false;
+
+		/**
```

Comment:
My bad, sorry. https://developer.joomla.org/coding-standards/inline-code-comments.html says `/*  */` whereas most projects use double ** 

```suggestion
		/*
```